src/main/scala/org/broadinstitute/dsde/workbench/leonardo/dns/ClusterDnsCache.scala

```diff
@@ -1,19 +1,23 @@
 package org.broadinstitute.dsde.workbench.leonardo.dns
 
 import akka.actor.{Actor, Props}
+import akka.http.scaladsl.model.Uri.Host
 import akka.pattern.pipe
 import com.typesafe.scalalogging.LazyLogging
 import org.broadinstitute.dsde.workbench.leonardo.config.ProxyConfig
 import org.broadinstitute.dsde.workbench.leonardo.db.DbReference
 import org.broadinstitute.dsde.workbench.leonardo.dns.ClusterDnsCache._
-import org.broadinstitute.dsde.workbench.leonardo.model.Cluster
-import org.broadinstitute.dsde.workbench.leonardo.model.ModelTypes.GoogleProject
+import org.broadinstitute.dsde.workbench.leonardo.model._
+
 import scala.concurrent.Future
 
 object ClusterDnsCache {
   // This is stored as volatile in the object instead of inside the actor because it needs to be
   // accessed by JupyterNameService. JupyterNameService is instantiated by the container and is
   // stateless, so it doesn't have an ExecutionContext, etc needed to interact with an Actor.
+
+  // TODO: should/can this use value classes like Map[Host, IP] ?
```

Comment:
Might as well? You'd need to also update `JupyterNameService` to wrap/unwrap the types I think.